spec/services/keybase_proof_service_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe KeybaseProofService, type: :service do
+  before do
+    base_url = 'https://keybase.io/_/api/1.0/sig/check_proof.json'
```

Comment:
why not use the config initializer? and put the `check_proof` endpoint there too

Reply:
why is the info repeated here?